experimental/generation/generator/packages/library/test/forms/unittest_phonenumber.schema

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "properties": {
+        "PhoneNumber": {
+            "type": "string",
+            "$entities": [
+                "phonenumber"
+            ]
```

Comment:
Can you add a phoneNumber.schema?  This means composer can discover it when building a form.